sources/packages/api/src/esdc-integration/e-cert-full-time-integration/e-cert-full-time-integration.service.ts

```diff
@@ -211,12 +215,92 @@ export class ECertFullTimeIntegrationService extends SFTPIntegrationBase<void> {
     } as CreateRequestFileNameResult;
   }
 
+  /**
+   * Get the list of all E-Cert response files from the
+   * MSFT-Response folder
+   * Filename pattern is `PEDU.PBC.CERTSFB.yyyymmdd.001`
+   * @returns full file paths for all response files from the
+   * E-Cert response folder.
+   */
+  async getResponseFilesFullPath(): Promise<string[]> {
```

Comment:
This method exists in the base class and it could be completely removed.

Reply:
Thanks for that, updated